bitnami/mysql/README.md

```diff
@@ -26,7 +26,7 @@ Bitnami charts can be used with [Kubeapps](https://kubeapps.com/) for deployment
 To install the chart with the release name `my-release`:
 
 ```bash
-$ helm repo add bitnami https://charts.bitnami.com/bitnami
+$ helm repo add stable https://kubernetes-charts.storage.googleapis.com
```

Comment:
Adding this change
```suggestion
$ helm repo add bitnami https://charts.bitnami.com/bitnami
```
and bumping the chart version in the `bitnami/mysql/Chart.yaml` file is enough to merge this PR.